### ISSUE TITLE
Entry Credit Balance Increases are always before minute 1

### ISSUE THREAD

**Issue by carryforward** [FactomProject/WorkItems#637]
In the Binaries, the Balance Increase messages always appear before minute marker 1.  They should appear in the same minute as the Factoid transaction they point to, and preferably in order with other EC items.

This was submitted after minute 2 ended:
```
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 0C 
8B 2F 3E 53 E9 2F 1B C8 F5 F9 D8 E8 D0 66 34 6A 35 69 81 C3 A8 BA 44 6D 5B 91 3A 81 90 B3 58 EE 
98 3A C0 8B C3 9C 81 6A EE E4 67 77 84 D7 DE AB B7 2E 11 73 73 CE 4A A1 3F 38 04 78 4F D3 68 B4 
E0 3D E7 66 1F E6 39 4C 41 0B 69 05 E0 4B B2 9E 42 B3 EC 26 04 4C 5D 0D AE 13 63 61 F4 54 05 30 
00 00 00 0A 
00 
00 00 00 00 00 00 00 0C 
00 00 00 00 00 00 00 59 
00 00 

04 
28 F0 C8 DC F9 74 34 0B 16 AF 79 D0 5D 84 C2 88 28 1A C1 CC C7 F0 FC 17 1E 65 C2 E7 8F EE AE 9C 
11 EC BA 0C C7 1E AD 03 4C F9 E5 FF F3 8E 9D A7 C6 25 8D 06 79 67 4B 33 DC 7E 94 62 7F B6 97 2C 
00 
00 

01 01 01 02 01 03 01 04 01 05 01 06 01 07 01 08 01 09 01 0A
```

notice the minute markers show it happened before minute 1 ended.

**Comment by carryforward** [comment]
Here is the factoid transaction for the purchase:

```
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 0F 
C6 8B D4 AD E8 5D 49 93 6E 3A 3B FE 63 C7 6F DD CC 27 97 1A 61 D5 21 C4 2E 60 16 0C 46 84 0C BD 
CA 03 B4 8E 91 A2 28 48 06 1A F3 64 1A 92 7F AD CA FA 0F 20 42 E8 5E FB 26 50 F2 D6 BE 62 DD D6 
93 0A 4B E9 B2 3C 5E A6 EC D3 8D D2 05 2B 4F FA F4 B1 DC B3 78 B8 6B 1E 0E 18 B5 78 4B B2 48 3E 
00 00 00 00 00 0A 2C 2A 
00 00 00 0A 
00 
00 00 00 02 
00 00 02 38 

02 01 4F 2A 1D 32 E1 00 0A 00 92 D0 97 E4 00 2A 2B 40 F6 A8 2E 14 95 63 CC E4 37 72 A9 E1 26 81 9D DC 40 63 44 CD DD 58 03 22 45 28 71 F9 22 92 D0 97 E4 00 30 E7 63 45 90 57 08 98 70 EC F2 16 33 25 62 2B 3E 08 E2 49 E9 0F 78 F0 DD F5 07 5A 7E 28 2E C9 92 D0 97 E4 00 9A D5 26 BF 1E 6E 27 5E AC 67 C3 2F C1 AB C3 AE 99 3F B2 BB B8 EA 3C 4E 68 CA 45 37 D9 A9 0F 87 92 D0 97 E4 00 15 A4 85 76 C8 09 D3 81 42 A4 FF 1D 55 5B F4 DE 56 55 D9 1F 38 D4 B7 31 17 3A B3 F6 80 28 F4 E3 92 D0 97 E4 00 DC EB 1C E5 77 84 44 E7 77 71 72 E1 F5 86 48 8D 23 82 FB 10 37 88 7C D7 9A 70 B0 CB A4 FB 3D CE 92 D0 97 E4 00 78 72 2C EB D9 DF 86 E9 8C 05 09 03 DE CD 4B E9 79 CA AD 56 E7 BF 1A A0 1A 3E 5A 66 2D 7D 21 8C 92 D0 97 E4 00 E1 05 F4 F9 65 42 93 32 A2 2E ED 63 75 3E 11 80 AE 7A 0F D0 92 56 39 5D FB 5A D2 D7 0E FA 09 C2 92 D0 97 E4 00 98 81 AE B2 64 45 2A 4F 7F AF A1 CC 7B C4 B9 3A 05 C5 55 37 C0 70 34 53 E5 85 F6 D8 3C E7 7D CA 92 D0 97 E4 00 43 7F DC FE C2 16 C8 6A 95 E6 69 C6 81 AD DA B1 7E D3 6B E3 2A C4 95 7F 84 A9 07 82 DB B4 77 16 92 D0 97 E4 00 B7 80 3B C7 9F 68 67 9D 23 EC 17 0C 57 0C D0 36 91 B9 FC BD E6 C2 A0 FE A7 64 67 09 17 9D 0C 2C 
^coinbase

00 00 
^two end of minute markers

\/ purchase transaction
02 
01 4F 2A 1D D8 A6 
01 00 01 
81 EE B5 CA 00 
59 7F A4 C6 29 A5 B1 74 8F 3F 58 E9 7B E4 99 C0 31 AB 29 5D F9 E7 41 05 DD DD 60 38 89 37 EB 8B 
CE 10 
28 F0 C8 DC F9 74 34 0B 16 AF 79 D0 5D 84 C2 88 28 1A C1 CC C7 F0 FC 17 1E 65 C2 E7 8F EE AE 9C 

\/rcd and sig
01 
9C E4 8A 27 0C 32 0C 04 2A 81 FC 81 FA 13 14 73 A7 9C CD 4A 4D EF 33 03 8B 75 C8 73 62 F9 F9 E8 
23 32 F8 06 08 D5 CB 2D EF B8 E4 71 BD 90 BF 56 79 0C 5A 56 89 C3 2E 95 91 7C D3 93 B6 FD 3A 65 
23 85 B2 43 60 53 D9 E2 D5 44 DF 14 86 67 89 A8 6B A2 AA E7 6E 61 A2 B4 34 0C DE 1B 2C E2 78 08 


\/ 8 more minute markers
00 00 00 00 00 00 00 00


```

**Comment by michaelbeam** [comment]
I believe I have fixed the issue. please test with 10 min blocks

**Comment by carryforward** [comment]
paul has a fix for this.

**Comment by carryforward** [comment]
Paul's fix didn't work.  the error is around line 355 of the processor and in processesbyentrycredit 647

**Comment by carryforward** [comment]
fixed, but may have caused #693
